apps/docs/pages/guides/ai/choosing-compute-addon.mdx

```diff
@@ -17,96 +17,49 @@ You have two options for scaling your vector workload:
+| ------ | --------- | --- | --------------- | --------- | ---- | ------------ | ----------- | ------------------ | ------ |
+| Free   | 15,000    | 16  | 40              | 40        | 480  | 0.011 sec    | 0.016 sec   | 1 GB + 200 Mb Swap | 1 GB   |
+| Small  | 50,000    | 32  | 64              | 100       | 175  | 0.031 sec    | 0.051 sec   | 2 GB + 200 Mb Swap | 2 GB   |
+| Medium | 100,000   | 32  | 64              | 100       | 240  | 0.083 sec    | 0.126 sec   | 4 GB               | 4 GB   |
+| Large  | 224,482   | 32  | 64              | 100       | 280  | 0.017 sec    | 0.028 sec   | 8 GB               | 8 GB   |
+| XL     | 500,000   | 24  | 56              | 100       | 360  | 0.055 sec    | 0.135 sec   | 13 GB              | 16 GB  |
+| 2XL    | 1,000,000 | 24  | 56              | 250       | 560  | 0.036 sec    | 0.058 sec   | 32 GB              | 32 GB  |
+| 4XL    | 1,000,000 | 24  | 56              | 250       | 950  | 0.021 sec    | 0.033 sec   | 39 GB              | 64 GB  |
+| 8XL    | 1,000,000 | 24  | 56              | 250       | 1650 | 0.016 sec    | 0.023 sec   | 40 GB              | 128 GB |
+| 12XL   | 1,000,000 | 24  | 56              | 250       | 1900 | 0.015 sec    | 0.021 sec   | 38 GB              | 192 GB |
+| 16XL   | 1,000,000 | 24  | 56              | 250       | 2200 | 0.015 sec    | 0.020 sec   | 40 GB              | 256 GB |
 
-| Plan   | Vectors   | Lists | RPS | Latency Mean | Latency p95 | RAM Usage | RAM    |
-| ------ | --------- | ----- | --- | ------------ | ----------- | --------- | ------ |
-| Free   | 20,000    | 40    | -   | -            | -           | -         | 1 GB   |
-| Small  | 50,000    | 100   | -   | -            | -           | -         | 2 GB   |
-| Medium | 100,000   | 200   | -   | -            | -           | -         | 4 GB   |
-| Large  | 250,000   | 500   | -   | -            | -           | -         | 8 GB   |
-| XL     | 500,000   | 1000  | -   | -            | -           | -         | 16 GB  |
-| 2XL    | 1,000,000 | 2000  | 140 | 0.358 sec    | 0.575 sec   | 30 GB     | 32 GB  |
-| 4XL    | 1,000,000 | 2000  | 270 | 0.186 sec    | 0.304 sec   | 35 GB     | 64 GB  |
-| 8XL    | 1,000,000 | 2000  | 470 | 0.104 sec    | 0.166 sec   | 38 GB     | 128 GB |
-| 12XL   | 1,000,000 | 2000  | 600 | 0.085 sec    | 0.132 sec   | 41 GB     | 192 GB |
-| 16XL   | 1,000,000 | 2000  | 670 | 0.081 sec    | 0.129 sec   | 45 GB     | 256 GB |
+Accuracy was 0.99 for benchmarks.
 
-For 1,000,000 vectors 40 probes results to precision of 0.98. Note that exact values may vary depending on the dataset and queries, we recommend to run benchmarks with your own data to get precise results. Use this table as a reference.
+QPS can also be improved by increasing `m` and `ef_construction`. This will allow you to use a smaller value for `ef_search` and increase QPS. For example, increasing `m` to 32 and `ef_construction` to 80 for 4XL will increase QPS to 1280.
```

Comment:
```suggestion
QPS can also be improved by increasing [`m` and `ef_construction`](/docs/guides/ai/going-to-prod#hnsw-understanding-efconstruction--efsearch--and-m). This will allow you to use a smaller value for `ef_search` and increase QPS. For example, increasing `m` to 32 and `ef_construction` to 80 for 4XL will increase QPS to 1280.
```